File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.js
function solution(seoul) {
    // forEach 는 undefined를 반환
    // find 는 해당하는 요소
    // findIndex는 해당하는 요소의 인덱스
    // 단 find는 요소의 첫 번째를 가져온다.
    
    const idx = seoul.findIndex((ele) => ele === "Kim");
    const str =`김서방은 ${idx}에 있다`;
    return str;
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        int answer = 0;
        for(int i = 0; i < seoul.length; i++){
            if(seoul[i].equals("Kim")){
                answer = i;
                break;
            }
        }
        return String.format("김서방은 %d에 있다",answer);
    }
}